src/System.Windows.Forms/src/System/Windows/Forms/DataGridView.SelectedCellsAccessibleObject.cs

```diff
@@ -8,11 +8,11 @@ public partial class DataGridView
 {
     private class DataGridViewSelectedCellsAccessibleObject : AccessibleObject
     {
-        private readonly DataGridView _ownerDataGridView;
+        private readonly DataGridViewAccessibleObject _ownerDataGridViewAO;
```

Comment:
We usually spell out abbreviations
```suggestion
        private readonly DataGridViewAccessibleObject _ownerDataGridViewAccessibleObject;
```

Reply:
Done.